test/run.sh

```diff
@@ -120,7 +120,7 @@ function run_integration_tests() {
 function run_acceptance_tests() {
   # for now we need to run the tests sequentially, there seems to be some sort of issues with running them in parallel
     for pkg in $(go list ./... | grep 'test/acceptance'); do
-      if ! go test -count 1 -race "$pkg"; then
+      if ! go test -count 1 "$pkg"; then
         echo "Test for $pkg failed" >&2
         return 1
       fi
```

Comment:
@dirkkul wdyt about changing this part to:
```
    for pkg in $(go list ./... | grep 'test/acceptance' | grep -v 'test/acceptance/stress_tests' ); do
      if ! go test -count 1 -race "$pkg"; then
        echo "Test for $pkg failed" >&2
        return 1
      fi
    done
    for pkg in $(go list ./... | grep 'test/acceptance/stress_tests' ); do
      if ! go test -count 1 "$pkg"; then
        echo "Test for $pkg failed" >&2
        return 1
      fi
    done
```
so that only `stress tests` would run explicitly without `-race`?